Source package : yoshimi
File name       : yoshimi-2.3.0.tar.bz2
Lookaside URL   : https://src.fedoraproject.org/repo/pkgs/yoshimi/yoshimi-2.3.0.tar.bz2/sha512/41fe5dc89582586224593f9ae0f6216e752618e4d43e369285dadeb0c57da96b0168e4752a0c2014aac1dcb77df46a921dc1fd9698f99101d95cb61c5bed0dc1/yoshimi-2.3.0.tar.bz2
Source URL      : http://downloads.sourceforge.net/yoshimi/yoshimi-2.3.0.tar.bz2

source file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/source-yoshimi-2.3.0.tar.bz2: bzip2 compressed data, block size = 900k
lookaside file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/lookaside-yoshimi-2.3.0.tar.bz2: bzip2 compressed data, block size = 900k


Only in source-yoshimi-2.3.0.tar.bz2-extracted/yoshimi-2.3.0: build
Only in lookaside-yoshimi-2.3.0.tar.bz2-extracted/yoshimi-2.3.0/dev_notes: MPE_spec.pdf
Binary files source-yoshimi-2.3.0.tar.bz2-extracted/yoshimi-2.3.0/doc/Histories.tar.bz2 and lookaside-yoshimi-2.3.0.tar.bz2-extracted/yoshimi-2.3.0/doc/Histories.tar.bz2 differ
diff -r source-yoshimi-2.3.0.tar.bz2-extracted/yoshimi-2.3.0/doc/Yoshimi_2.3.0_features.txt lookaside-yoshimi-2.3.0.tar.bz2-extracted/yoshimi-2.3.0/doc/Yoshimi_2.3.0_features.txt
10c10
< These files are plain text and laid out to be particularly easy to edit. The 'classic' one is actually a reference file auto-generated by Yoshimi. Rather than changing this you should make a renamed copy and alter that. Almost every element can be edited, and changes take place immediately the theme is selected/reselected. There are Import and Export options available so you can easily export a theme, change it in any text editor then import it again, optionally with a different name.
---
> These files are plain text and laid out to be particulalry easy to edit. The 'classic' one is actually a reference file auto-generated by Yoshimi. Rather than changing this you should make a renamed copy and alter that. Almost every element can be editied, and changes take place immediately the theme is selected/reselected. There are Import and Export options available so you can easily export a theme, change it in any text editor then import it again, optionally with a different name.
Only in source-yoshimi-2.3.0.tar.bz2-extracted/yoshimi-2.3.0/src: version.txt
